CoffeeChat/Controller/MatchViewController.swift

```diff
@@ -9,16 +9,75 @@
 import GoogleSignIn
 import UIKit
 
+enum ChatStatus {
+    /// No one has reached out yet
+    case planning
+    /// No one has reached out in 3 days
+    case noResponses
+
+    /// User already reached out and is waiting on pair
+    case waitingOn(SubUser)
+    /// Pair already reached out, and user is responding
+    case respondingTo(SubUser)
+
+    /// The chat date has passed
+    case finished
+    /// The chat has been cancelled
+    case cancelled(SubUser)
+
+    /// Chat has been scheduled and is coming up
+    case chatScheduled(SubUser, Date)
+
+    // TODO change when matching responses change
+    // The current Matching doesn't hold enough information to distinguish between all states, so this function will
+    // change in the near future
+    static func forMatching(matching: Matching) -> ChatStatus {
+        let matchDaySchedule = matching.schedule.first
+        let matchPear = matching.users[1]
+        if let matchDaySchedule = matchDaySchedule, matching.active {
```

Comment:
Is this always non nil?

Reply:
I was using a default value for testing until the `Matching` gets finalized